src/main/java/com/milaboratory/mixcr/cli/ActionAssembleContigs.java

```diff
@@ -81,8 +86,13 @@ public void go(ActionHelper helper) throws Exception {
                 for (Clone cl : clones)
                     tmpOut.writeObject(cl);
             }
+
+            assert report.getInitialCloneCount() == reader.numberOfClones();
         }
 
+        assert report.getFinalCloneCount() == totalClonesCount;
```

Comment:
? should use equality with error, e.g. Math.abs(a - b) < 1e-6 ?

Reply:
This is number of clones (`int`)

Reply:
😳👌